functions/Find-DbaOrphanedFile.ps1

```diff
@@ -268,11 +253,9 @@ Finds the orphaned ending with ".fsf" and ".mld" in addition to the default file
 
 		}
 	}
-	END
-	{
-		if ($result.count -eq 0)
-		{
-			Write-Output "No orphaned files found"
+	end {
+		if ($result.count -eq 0) {
+			Write-Message -Level Output -Message "No orphaned files found"
```

Comment:
great rewrite! This particular one i'd like moved up to the server loop and changed into verbose.